bftengine/tests/bcstatetransfer/bcstatetransfer_tests.cpp

```diff
@@ -37,14 +37,28 @@ using namespace impl;
 
 // Create a test config with small blocks and chunks for testing
 Config TestConfig() {
-  Config config;
-  config.myReplicaId = 1;
-  config.fVal = 1;
-  config.numReplicas = 4;
-  config.maxBlockSize = kMaxBlockSize;
-  config.maxChunkSize = 128;
-  config.maxNumberOfChunksInBatch = 128;
-  return config;
+  return {1,
```

Comment:
This is next to impossible to read. Can you please change it back from uniform initialization so that we can tell what is actually being configured?

Reply:
Just put comments, now it's clear.
It's a unittest after all.

Reply:
Thank you!